source/test-suite/Main.hs

```diff
@@ -350,7 +350,7 @@ main = Hspec.hspec . Hspec.parallel . Hspec.describe "cabal-gild" $ do
   Hspec.it "formats a comment after a field's value" $ do
     expectGilded
       "f:\n 1\n -- c"
-      "f: 1\n-- c\n"
+      "f:\n  1\n\n-- c\n"
```

Comment:
This expectation had to change because the input had the field's value on another line:

``` cabal
f:
 1
 -- c
```

The intent of the test was to make sure that comments that appear to be "inside" a field are captured even when they're actually not part of the field at all. So this new expectation covers that behavior.